### ISSUE TITLE
Bug with accumulate function in train.py

### ISSUE THREAD

**Issue by ParthaEth** [rosinality/stylegan2-pytorch#172]
The accumulate function found in train.py -https://github.com/rosinality/stylegan2-pytorch/blob/master/train.py#L49. Only accumulates running average of optimiser trainable parameters fo the generator model. If the generator has parameters that are not trained by optimiser such as the ones found in batch_norm layers, then this function will not accumulate those parameters. Stylegan model specifically does not have those kind of parameters so for this particular project this function is valid but should not be copied over to another project.

**Comment by rosinality** [comment]
Yes, and you should be careful on using ema on buffers like running statistics in the batch norm.
